src/modules/apis.js

```diff
@@ -0,0 +1,132 @@
+const baseUrl = 'https://api.spacexdata.com/v3/rockets';
+const likeBaseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5GbgWWZPMVcYokoIs4ID/likes';
+const commentBaseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+
+const appId = '5GbgWWZPMVcYokoIs4ID';
+
+// get all rockets 🚀🚀
+const getAllRockets = async () => {
+  try {
+    const response = await fetch(`${baseUrl}`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+    });
+    if (!response.ok) {
+      throw new Error('Could not fetch data');
+    }
+    const responseData = await response.json();
+
+    // the responese data is an array so loop through and extract the relevant data needed
+    const rocketsData = responseData.map(({
+
+      rocket_name, rocket_id, flickr_images, cost_per_launch,
+      success_rate_pct, mass, rocket_type,
+    }) => ({
+      rocket_type,
+      mass,
+      success_rate_pct,
+      cost_per_launch,
+      rocket_name,
+      rocket_id,
+      flickr_images: flickr_images[0], // Extract the first element from the flickr_images array
+    }));
+
+    localStorage.setItem('rocketData', JSON.stringify(rocketsData));
+    return rocketsData;
+  } catch (error) {
+    /* eslint-disable no-console */
+    console.log(error.message);
+    /* eslint-enable no-console */
```

Comment:
## Optional Changes ♻️

I noticed that the previous reviewer did not pick this up:

- [ ] In your project requirements, if your click the JavaScript Link for JavaScript Best practices and rules, you're asked to not commit `console.log` to your repo. Because the previous reviewer did not pick this up, I'm marking these changes as optional but I highly recommend you remove all console.log statements in your codebase as this will affect you in your next milestone. Please consider removing them and not committing any console.log statement  👍🏽
- [ ] Kindly remove the console logs in all the other files. I decided to comment only on this one so I don't fill the PR with the same feedback, but you're having them in many places. Please remove them. 👍🏽

<img width="1158" alt="Screenshot 2023-05-27 at 7 31 14 PM" src="https://github.com/til2to/javascript-capstone/assets/50941074/6c01a4fe-b611-4a83-a51a-e0b9b9e9602c">

Reply:
Noted